src/constants/networks.ts

```diff
@@ -95,11 +97,24 @@ export const BNBNetworkInfo: NetworkInfo = {
   blurb: '',
 }
 
+export const AvalancheNetworkInfo: NetworkInfo = {
+  chainId: 43114,
+  id: SupportedNetwork.AVALANCHE,
+  route: 'avalanche',
```

Comment:
in the UX we currently have the route as `avax` for a full url of `https://info.uniswap.org/#/avax/`

Reply:
cool ill change to this, and good callout on the "L2" will remove the badge